### PR TITLE
#2053 - Invalid Education Program Field of Study

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Added the missing W3 combination.
- Added required message to avoid the "Internal server error" from API in the future if some other combination is missing.
- Removed the non longer needed `applicationId` and `applicationStatus` hidden fields.

The previous combination that was failing is now correctly setting the field of study as number `33`.
image

Simulated behavior for when a combination is missing.
image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/educationprogram.json [hunk 1]
```diff
@@ -864,13 +863,13 @@
...
                     "validate": {
-                        "required": false,
+                        "required": true,
                         "minLength": "",
                         "maxLength": "",
                         "minWords": "",
                         "maxWords": "",
                         "pattern": "",
-                        "customMessage": "",
+                        "customMessage": "The combination of the Credential Type and CIP did not result in a known Field of Study code.",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/educationprogram.json:872]: 👍

**Review by dheepak-aot — APPROVED** [review]
Thanks for taking care of the fix. 👍

**Review by andrepestana-aot — APPROVED** [review]
👍

**Review by sh16011993 — APPROVED** [review]
👍

**Review by ann-aot — APPROVED** [review]
👍

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrewsignori-aot

**Pull request merged**